Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM alpine:3.9 as builder
+
+RUN set -xe; \
+    apk add --no-cache nodejs python3 supervisor yarn xmlsec netcat-openbsd postgresql-client; \
+    pip3 install --upgrade pip
+
+RUN wget -O powerdns-admin.tar.gz https://github.com/timja/PowerDNS-Admin/archive/6b1282e710a4031659ed340ca7ef0518b4893f89.tar.gz \
+    && tar -xzf powerdns-admin.tar.gz \
+    && mv PowerDNS-Admin-* /powerdns-admin/ \
+    && rm powerdns-admin.tar.gz
+
+WORKDIR /powerdns-admin
+
+RUN set -xe; \
+    apk add --no-cache --virtual _build alpine-sdk libffi-dev xmlsec-dev python3-dev openldap-dev mariadb-connector-c-dev postgresql-dev; \
```

Comment:
Why do you need 2 apk layers?

Reply:
This is a virtual one and is only needed to install the requirements, the others are needed at runtime